backend.py

```diff
@@ -239,6 +245,23 @@ def apply_effect(self, robot, state):
         """
         robot.walk(self.distance, state)
 
+    def as_dict(self):
+        """
+        Return card´s info as dictionary for sending with server.
+        """
+        return {"MovementCard":
```

Comment:
tady mám dotaz, jestli tu nemá být i to self.name karty?

Reply:
Je to někde potřeba?

Reply:
Já nevím, Netuším pořádně jak se ty karty budou posílat a co je potřeba, spíš si myslím že asi ne.